test/pages/calendar.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import moment from 'moment';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import CalendarPage from '../../src/pages/calendar';
+import '../__mocks__/intersectionObserverMock';
+import { CalendarEvent } from '../../src/types';
+
+const firstStart = moment();
+const secondStart = moment();
+const thirdStart = moment();
+const fourthStart = moment();
+
+const mockEvents: CalendarEvent[] = [
```

Comment:
These mock data should be removed, right/

Reply:
No it’s needed for rendering the calendar and asserting in the tests

Reply:
Oops, sorry missed the file name